tests/resources/db/latest_schema.sql

```diff
@@ -124,7 +125,9 @@ CREATE TABLE runs (
 	CONSTRAINT run_pk PRIMARY KEY (run_uuid),
 	FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
 	CONSTRAINT runs_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted')),
+	FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
 	CONSTRAINT source_type CHECK (source_type IN ('NOTEBOOK', 'JOB', 'LOCAL', 'UNKNOWN', 'PROJECT')),
+	FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
```

Comment:
These are duplicates.

Reply:
https://github.com/sqlalchemy/sqlalchemy/issues/11677 seems related?

Reply:
Smells like a bug in the release. Pin for now?